2021-12-16 20:00:55.347257 | API is disabled on startup
2021-12-16 20:00:55.348257 | Settings loaded!
2021-12-16 20:00:56.118649 | Started repair process...
2021-12-16 20:00:56.120651 | Propting user to select VRCA...
2021-12-16 20:01:02.981137 | VRCA selected: C:/Users/adamc/Downloads/Avatar-Pandas-Demon-Compendium-.file_e3f724e6-61fd-4aec-8ec5-a0e47e5e27c7.2_1.vrca
2021-12-16 20:01:02.982139 | Starting repair on new thread...
2021-12-16 20:01:02.982139 | Error occured during repair process!
Traceback (most recent call last):
  File "C:\Users\adamc\Documents\GitHub\A.R.E.S\ARES(rewritten)\GUI.py", line 229, in RepairVRCAWrapper
    threading.Thread(target=RepairVRCA, args={self.RepairVRCA}).start()
NameError: name 'RepairVRCA' is not defined

